scrapper.py

```diff
@@ -44,18 +77,72 @@ def get_search_urls(self):
         pass
```

Comment:
according to docstring, this method is supposed to return seed_urls value

Reply:
Done it, if I understood it correctly